Vic2ToHoI4/Source/HOI4World/HoI4FocusTree.cpp

```diff
@@ -573,14 +567,10 @@ void HoI4FocusTree::addDemocracyNationalFocuses(const HoI4::Country& home,
 		auto warPlanFocus = GetLoadedFocus("WarPlan").makeTargetedCopy(home.getTag(), tag, hoi4Localisations);
 		warPlanFocus->updateFocusElement(warPlanFocus->bypass, "$TARGET", tag);
 		warPlanFocus->xPos = relativePos;
-		if (truceUntil)
-		{
-			warPlanFocus->updateFocusElement(warPlanFocus->available, "#TRUCE", "date > " + truceUntil->toString());
-		}
-		else
-		{
-			warPlanFocus->removePlaceholder(warPlanFocus->available, "#TRUCE");
-		}
+		warPlanFocus->updateFocusElement(warPlanFocus->available,
+			 "#TRUCE",
+			 "date > " + truceUntil->toString(),
+			 !!truceUntil);
```

Comment:
Why the double negation?

Reply:
Because `truceUntil` is std::optional

Reply:
Ah. It's less succinct, but could you instead do `truceUntil.has_value() && *truceUntil` just for clarity?

Reply:
Would `truceUntil.has_value()` be enough? Because `*truceUntil` is a date value that won't mix with `bool`

Reply:
Yes, that would be perfect. I had misunderstood what you were intending to check.